CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## v1.5.0
```

Comment:
Discussed with @lyuboxa, been a while since we last did a release, so gonna kick this off by adding a new tag and invoking our github actions

Reply:
Ah I see! I had the tag process mixed up. Sounds good, thanks!